x-pack/plugin/fleet/src/main/java/org/elasticsearch/xpack/fleet/rest/RestFleetMultiSearchAction.java

```diff
@@ -81,23 +81,25 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
             allowExplicitIndex,
             searchUsageHolder,
             clusterSupportsFeature,
-            (key, value, searchRequest) -> {
-                if ("wait_for_checkpoints".equals(key)) {
-                    String[] stringWaitForCheckpoints = nodeStringArrayValue(value);
+            (params, searchRequest) -> {
```

Comment:
Changing from the existing TriFunction for a BiConsumer for adding parameters to the request